scripts/Tools/Makefile

```diff
@@ -813,9 +813,8 @@ test_esmf: test_esmf.o
 #-------------------------------------------------------------------------------
 # create list of component libraries - hard-wired for current ccsm components
 #-------------------------------------------------------------------------------
-CLMVER = $(filter $(CLM_CONFIG_OPTS), clm5_0 clm4_5)
 ifeq ($(CIME_MODEL),cesm)
-  ifneq ($(CLMVER),$(null))
+  ifeq ($(COMP_LND),clm)
```

Comment:
I *think* we still need some logic in the makefile so that the correct thing is done for dlnd/slnd/xlnd. @jedwards - Does this change to the Makefile, together with the one below, look correct? I have tested cases both with CLM and with DLND in CESM and confirmed that it looks like the behavior is the same as before in both cases, but I'd be happy to get a critical set of eyes on the changes in this file.